src/components/collaborator_list/CollaboratorAddEdit.tsx

```diff
@@ -48,7 +50,7 @@ export default function CollaboratorAddEdit(props: CollaboratorAddEditProps): Re
         <div className='form-group row no-margin'>
             <div className='col-lg-12 col-md-12 col-sm-12 col-xs-12 collaborator-form-card' key={`collaborator-item-id`}>
                 <div className='row'>
-                    <h2>{collaborator === undefined ? `New ${collaboratorText} Information` : `Edit ${collaborator.name} Information`}</h2>
+                    <h2>{collaborator?.name === undefined ? `New ${collaboratorText} Information` : `Edit ${collaborator.name} Information`}</h2>
```

Comment:
Can you explain why this is required? Also, shouldn't `collaborator.name` always be a required field (and so the `?` is not necessary)?